### PR TITLE
Capture target text on autotrack events

### PR DESCRIPTION
**Pull request by jmtaber129**
Determine target text by concatenating target texts of component children, and include it in the autotracked event.

### CODE REVIEW

File: js/Heap.js [hunk 1]
```diff
@@ -62,15 +62,42 @@ export default Package.create({
...
           touchState,
+          targetText,
         });
       },
     };
   },
 });
 
+// :TODO: (jmtaber129): Consider implementing sibling target text.
+const getTargetText = fiberNode => {
```
Comment (drob) [inline js/Heap.js]: If the target text is empty, we're returning `''` which will be a little strange downstream in our UI. I propose we instead set it to null and filter out the prop from the blob entirely, in the `track` call. Wdyt?

Comment (drob) [reply js/Heap.js]: Oh, I see – you're using the empty but non-null string for purposes of recursion. We could do something outside of here in the caller to not send `''` to the server. Thoughts?

Comment (jmtaber129) [reply js/Heap.js]: Yea, when there's no target text on the web, that prop is just not sent.  Updated to only include that prop when it's non-empty

**Comment by drob** [comment]
LGTM

**Pull request merged**